src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -64,7 +64,7 @@ manualRegistrationToolBox::manualRegistrationToolBox(QWidget *parent) : medAbstr
     connect(d->b_startManualRegistration,SIGNAL(clicked()),this,SLOT(startManualRegistration()));
     d->b_startManualRegistration->setObjectName("startManualRegistrationButton");
 
-    QLabel* explanation = new QLabel("To add a landmark: \n\tShift + left mouse button\n", widget);
+    QLabel* explanation = new QLabel("To add a landmark: \n\tShift + left mouse button\nTo remove a landmark duo: \n\tBackspace + left mouse button", widget);
```

Comment:
Ah ok :D I read: "To remove a landmark do: blah blah" and thought it was a typo.
Maybe it would be clearer if it said "to remove a pair of landmarks" ?
What happens though if you try to remove a landmark that does not have a pair yet ?